marlowe-playground-client/src/Marlowe/Holes.purs

```diff
@@ -521,6 +543,41 @@ instance boundIsMarloweType :: IsMarloweType Bound where
 instance boundHasMarloweHoles :: HasMarloweHoles Bound where
   getHoles (Bound a b) m = m
 
+data ExtendedTimeout
```

Comment:
Why ExtendedTimeout and not just Timeout?

Reply:
snap!

Reply:
Good point